### PR TITLE
Improve passing test

### PR DESCRIPTION
**Pull request by big-kahuna-burger**
🕓

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -131,17 +128,25 @@ tape('should handle \'NotBeforeError\'', function (t) {
...
 
   servertest(createService(auth), '/', opts, function (err, res) {
     t.ifError(err, 'should not error')
     var data = JSON.parse(res.body)
 
     t.equal(data.email, 'chet@scalehaus.io', 'should have correct email')
-    t.equal(data.expiresIn, '30d', 'should have correct expiresIn')
-    t.ok(data.iat, 'chet@scalehaus.io', 'should have iat')
+    var expectedExpiry = now + 2592000
+    t.equal(data.exp, expectedExpiry, 'should have correct expiresIn')
```
Comment (davidguttman) [inline test/index.js]: Why was this changed from `expiresIn` to `exp`?

Comment (big-kahuna-burger) [inline test/index.js]: Since the expiry date has beed added trough sign opts, then `jsonwebtoken` package adds this as a standard jwt `exp` claim, where before it was only added as payload (not any different than email claim for example)
